src/lazyload-image.ts

```diff
@@ -30,11 +30,27 @@ export function isVisible(element: HTMLElement, threshold = 0, _window = window)
     );
 }
 
-function loadImage(imagePath: string): Observable<HTMLImageElement> {
+function loadImage(element: HTMLElement, imagePath: string, useSrcset: boolean, sourcesAttrName: string): Observable<HTMLImageElement> {
```

Comment:
`useSrcset` can be `undefined` here (if the user doesn't set the property) and even though it is falsely I think it is more clear to default it to false:
```ts
loadImage(element: HTMLElement, imagePath: string, sourcesAttrName: string, useSrcset = false): Observable<HTMLImageElement> {
```